yacc/console/lx_test.go

```diff
@@ -24,7 +24,7 @@ func TestSimpleLex(t *testing.T) {
 		},
 		{
 			query: "kill client 0xc00030f520;",
-			exp:   []int{spqrparser.KILL, spqrparser.IDENT, spqrparser.IDENT},
+			exp:   []int{spqrparser.KILL, spqrparser.CLIENT, spqrparser.IDENT},
```

Comment:
maybe we can add test on new token types(introduces in this pr?)

Reply:
you mean so, that expected will be []int{spqrparser.KILL, spqrparser.IDENT, spqrparser.IDENT} ?

Reply:
> maybe we can add test on new token types(introduces in this pr?)

you mean so, that expected will be []int{spqrparser.KILL, spqrparser.IDENT, spqrparser.IDENT} ?